docker-compose.yml

```diff
@@ -1,17 +1,17 @@
-# version: "3.7"
+version: "3.7"
 
-# services:
-#   web:
-#     build: .
-#     environment:
-#       NODE_ENV:
-#       MONGODB_URI: mongodb://mongo:27017/nodegoat
-#     command: sh -c "until nc -z -w 2 mongo 27017 && echo 'mongo is ready for connections' && node artifacts/db-reset.js && npm start; do sleep 2; done"
-#     ports:
-#       - "4000:4000"
+services:
+  web:
+    build: .
```

Comment:
#### Automated PR Comment From coverity
<!-- bac11bda3fc6a7717735efcbb2ad195f -->
<!-- Comment managed by Git Commenter, do not modify! -->
### Coverity Issue - Container privilege escalation allowed
_Low CWE-269_
The Docker Compose file sets the `no-new-privileges` attribute to `false` or omits it as the default value is `false`. This doesn't restrict the container from acquiring additional privileges via SUID or SGID bits. This allows privilege escalation, meaning a user can set the security context of the container and can perform root-level operations.
### How to fix
Disable container privilege escalation by explicitly setting `no-new-privileges` to `true`.